tests/Support/SupportCollectionTest.php

```diff
@@ -978,6 +988,12 @@ public function testExcept()
         $this->assertEquals(['first' => 'Taylor', 'email' => 'taylorotwell@gmail.com'], $data->except('last')->all());
     }
 
+    public function testExceptSelf()
```

Comment:
Are all of these new tests just things that got introduced since @besologic pushed his last build?